src/test/cls_rgw_gc/test_cls_rgw_gc.cc

```diff
@@ -260,22 +260,14 @@ TEST(cls_rgw_gc, gc_queue_ops3)
   uint64_t max = 2;
   bool expired_only = false, truncated;
   cls_rgw_gc_queue_list_entries(ioctx, queue_name, marker, max, expired_only, list_info1, &truncated, next_marker);
-  ASSERT_EQ(2, list_info1.size());
+  ASSERT_EQ(1, list_info1.size());
 
-  int i = 0;
-  for (auto it : list_info1) {
-    std::cerr << "[          ] list info tag = " << it.tag << std::endl;
-    if (i == 0) {
-      ASSERT_EQ("chain-1", it.tag);
-    }
-    if (i == 1) {
-      ASSERT_EQ("chain-0", it.tag);
-    }
-    i++;
-  }
+  auto it = list_info1.cbegin();
+  std::cerr << "[          ] list info tag = " << it->tag << std::endl;
+  ASSERT_EQ("chain-1", it->tag);
```

Comment:
@pritha-srivastava how is this exercised? after looking around, i found that it was neither registered as a unit nor is used by any qa suite.